internal/install/static_terraform_deployer_yml.go

```diff
@@ -0,0 +1,20 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package install
+
+const terraformDeployerYml = `version: '2.3'
+services:
+  terraform:
+    build: .
+    tty: true
+    environment:
+      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID} # TODO Extract to the env-file
```

Comment:
Do you plan to address this TODO in this PR or a follow up PR? If it's for a follow up PR, can you make an issue for it so we don't lose track of it and post the link to the issue in the comment?

Reply:
https://github.com/elastic/elastic-package/issues/235